src/v2/view-builder/views/symantec/AuthenticatorSymantecView.js

```diff
@@ -0,0 +1,46 @@
+import { _, loc } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
+
+const Body = BaseForm.extend({
+
+  title () {
+    return this.options.isChallenge
+      ? loc('oie.authenticator.challenge.title', 'login', [this.options.displayName])
+      : loc('oie.authenticator.enroll.title', 'login', [this.options.displayName]);
+  },
+
+  subtitle () {
+    return this.options.isChallenge
+      ? loc('oie.symantecVip.challenge.description', 'login')
+      : loc('oie.symantecVip.enroll.description', 'login');
+  },
+
+  save () {
+    return this.options.isChallenge
+      ? loc('mfa.challenge.verify', 'login')
+      : loc('mfa.enroll', 'login');
+  },
+
+});
+
+export default BaseAuthenticatorView.extend({
+  initialize () {
+    BaseAuthenticatorView.prototype.initialize.apply(this, arguments);
+
+    const currentAuthenticator = this.options.appState.get('currentAuthenticator');
+    const authenticatorEnrollments = this.options.appState.get('authenticatorEnrollments')?.value || [];
+    const existingAuthenticator = _.find(authenticatorEnrollments, { id: currentAuthenticator.id });
+
+    // If the authenticator is already enrolled, we're in a challenge flow as opposed to enrollment
+    this.options.isChallenge = existingAuthenticator || false;
+    this.options.displayName = currentAuthenticator.displayName;
+
+    this.Footer = this.options.isChallenge
+      ? AuthenticatorVerifyFooter
+      : AuthenticatorEnrollFooter;
+  },
+  Body,
```

Comment:
nit:
we really shall use same pattern creating Footer to create Body.

Reply:
Agree! Interestingly, the `AuthenticatorEnrollFooter` and `AuthenticatorVerifyFooter` are equivalent, rendering the exact same view. I'm still trying to track down why we have two implementations instead of a general one - but do believe we can safely generalize this into a single view.

If we want to adopt a pattern where we're moving more logic into the `BaseAuthenticatorView`, then we can rename this to `AuthenticatorFooter` and include it within the `BaseAuthenticatorView` by default. This'll cleanup a lot of duplicate code in our views!

I'll create a ticket to track this, in addition to bring it up within our channel to capture more feedback.

Reply:
- Good catch on the Footers.
- For the Body, I meant adding something like this to init function: `this.Body = this.option.isChallenge ? VerifyForm : EnrollForm`

Reply:
Ah - I see what you mean.

This is actually a pattern I'm advocating we [move away from](https://github.com/okta/okta-signin-widget/pull/1707/files/f89ce3ec2b0919cc027282a1bc344b8b67ec8e30#r585114176). Two forms are not always necessary and create duplicate code for a few of our authenticators.

I'll open up a draft PR that condenses these sometime this week to help visualize this.

Reply:
Cannot see much duplication. we can chat in your draft PR later.